modules/admin_manual/pages/configuration/server/virus-scanner-support.adoc

```diff
@@ -377,3 +388,117 @@ Scanner output rules are used in daemon/socket mode.
 . A file size limit applies both to background jobs and to file uploads.
 . After installing ClamAV and the related tools, you will have two configuration files: `/etc/freshclam.conf` and `/etc/clamd.d/scan.conf`. 
 . We recommend that you enable verbose logging in both `clamd.conf` and `freshclam.conf` until you get any kinks with your ClamAV installation worked out
+
+
+== ICAP
+
+{icap-url}[ICAP] is an open standard supported by many antivirus products. With the release of the _Anti-Virus_ app 1.0.0, other virus scanners beside ClamAV can be used via ICAP if you are running it on an ownCloud Enterprise Edition. Currently the only supported virus scanners are _Kaspersky ScanEngine_ and _McAfee Antivirus_ although far more products might simply work.
+
+=== Installation
+
+- Install the Anti-Virus app 1.0.0 from the {anti-virus-app-url}[ownCloud marketplace]. Alternatively, use this occ command:
+
+[source,console,subs="attributes+"]
+----
+{occ-command-example-prefix} market:install files_antivirus
+----
+
+- Enable the app as admin in ownCloud under Settings > Apps or with the following occ command:
+
+[source,console,subs="attributes+"]
+----
+{occ-command-example-prefix} app:enable files_antivirus
+----
+
+
+=== General Configuration
+
+ICAP ist configured via occ commands. Go to your ownCloud directory, usually `/var/www/owncloud`, and enter the following occ commands:
```

Comment:
ist -> is

Reply:
and this is miss leading.

icap settings are setup just like any other antivirus settings: there is an admin ui for it.

in addition all these settings values can be set using the occ command

Reply:
@DeepDiver1975 More than happy to do so. However, should we point admins to the config file: /opt/kaspersky/ScanEngine/etc/kavicapd.xml and tell them to restart kavicapd after they made changes?

Reply:
that is all kav and environment specific. no idea how people install kav in their environment.
kav supports various deployment mechanisms ... we don't want to document that all .... just drop that all from my pov

Reply:
Okay, thanks, config file is completely gone.